src/components/BookForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
```

Comment:
![Screen Shot 2020-07-14 at 12 03 19 AM](https://user-images.githubusercontent.com/40712459/87361808-f077b580-c564-11ea-9eb8-5b4906590021.png)

should be in folder `/sc/containers`